pandas/io/parsers.py

```diff
@@ -825,6 +825,22 @@ def _validate_usecols_arg(usecols):
     return usecols
 
 
+def _validate_parse_dates_arg(parse_dates):
```

Comment:
so you are check scalars, just check that it is a ``list`` or ``dict``, that would be good enough for now. prob don't have a tests with passing something that is not a list/doc/scalar